target/linux/mpc85xx/files/arch/powerpc/boot/dts/hiveap-370.dts

```diff
@@ -0,0 +1,294 @@
+/*
+ * Aerohive HiveAP-370 Device Tree Source
+ *
+ * Copyright (C) 2017 Chris Blake <chrisrblake93@gmail.com>
+ *
+ * This program is free software; you can redistribute  it and/or modify it
+ * under  the terms of  the GNU General  Public License as published by the
+ * Free Software Foundation;  either version 2 of the  License, or (at your
+ * option) any later version.
+ */
+
+/include/ "fsl/p1020si-pre.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/leds/common.h>
+/ {
+	model = "Aerohive HiveAP-370";
+	compatible = "aerohive,hiveap-370";
+
+	aliases {
+		led-boot = &led_status;
+		led-failsafe = &led_status;
+		led-running = &led_status;
+		led-upgrade = &led_status;
+		label-mac-device = &enet0;
+	};
+	
+	leds {
+		compatible = "gpio-leds";
+		led_status: status {
+			label = "ap370:white:status";
+			gpios = <&gpio0 11 GPIO_ACTIVE_HIGH>;
+			default-state = "on";
+			linux,default-trigger = "default-on";
```

Comment:
Does that trigger exist?

Reply:
From what I can see, it does exist:

root@OpenWrt:~# cat /sys/class/leds/ap370\:white\:status/trigger
[none] timer default-on netdev phy0rx phy0tx phy0assoc phy0radio phy0tpt phy1rx phy1tx phy1assoc phy1radio phy1tpt

Reply:
Interesting. I'm used to just using the default-state property here.